content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -141,15 +194,15 @@ docker run \--detach -v "/DockerDir:/data" -p 9004:80 -p 9005:443 ixsystems/true
 {{< hint info >}}
 Use [Windows compatible syntax](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats) when specifying paths in the Windows file system.
 For example, if the created directory for the TrueCommand image was created in the Windows <file>Documents</file> folder, the `docker` command would be:
-`docker run \--detach -v C:\Users\\Example\\Documents\\DockerDir`.
-In this command *C* is the drive letter, *Example* is the current user name, and *DockerDir* is the TrueCommand image directory.
+`docker run \--detach -v {C}:\Users\\{Example}\\Documents\\{DockerDir}`.
+`{C}` is the drive letter, `{Example}` is the current user name, and `{DockerDir}` is the TrueCommand image directory.
 {{< /hint >}}
 
-Although there are different ways to run a Docker container, `-v /*hostdirectory*:/data` is required for TrueCommand to function.
+Although there are different ways to run a Docker container, `-v /{hostdirectory}:/data` is required for TrueCommand to function.
```

Comment:
is hostdirectory a variable?

Reply:
Yes, but italics won't work inside a code block.